src/Service/ElementService.php

```diff
@@ -27,8 +27,14 @@ public function __construct()
 		$this->pageRepository = Container::get(PageRepository::class);
 	}
 
-	public function addAdminMenu(): void
-	{
+    public function register(): void
```

Comment:
Přejmenovat register  na registerUserTableColumn nebo něco takového, aby to bylo trochu přehlednější